packages/gui/src/components/notification/NotificationsDropdown.tsx

```diff
@@ -33,7 +33,7 @@ export default function NotificationsDropdown() {
           sx={buttonStyle}
         >
           <Badge color="primary" badgeContent={unseenCount} invisible={!unseenCount}>
-            <NotificationsNoneIcon color="secondary" />
+            <NotificationIcon color="secondary" />
```

Comment:
secondary color inside the theme means "#000", but Figma has #757575 for basically all the icons in top section of the page

Reply:
also svg inherits color="secondary", so if we remove color="secondary" only in the icon, it will still stay black :/

Reply:
I'll create an issue to track updating the theme with a "headerIcon" type or similar.